src/libstore/unix/user-lock.cc

```diff
@@ -74,8 +74,13 @@ struct SimpleUserLock : UserLock
             debug("trying user '%s'", i);
 
             struct passwd * pw = getpwnam(i.c_str());
-            if (!pw)
-                throw Error("the user '%s' in the group '%s' does not exist", i, settings.buildUsersGroup);
+            if (!pw) {
+                printError("error: the user '%s' in the group '%s' does not exist", i, settings.buildUsersGroup);
```

Comment:
Does `printError` not already include the `error:` prefix?

Reply:
No. at least I'm not seeing a duplication during testing. This should set the error level appropriately, but still allow the requested build action to succeed. And give a note how to address the problem (might need a better note)